### ISSUE TITLE
#5 (AssertationError)

### ISSUE THREAD

**Issue by aczy118** [prajjwal1/autonomous-object-detection#7]
Hi again,

Unfortunately, I could not solve the issue. Even downgrading pytorch version to 1.1 and torchvision to 0.3 did not change anything.  I think this:

if targets is not None:
            for t in targets:
                # TODO: https://github.com/pytorch/pytorch/issues/26731
                floating_point_types = (torch.float, torch.double, torch.half)
                assert t["boxes"].dtype in floating_point_types, 'target boxes must of float type'
                **assert t["labels"].dtype == torch.int64, 'target labels must of int64 type'**
                if self.has_keypoint():
                    assert t["keypoints"].dtype == torch.float32, 'target keypoints must of float type'

is the part giving this error and it is in the roi_heads.py which I did not create it but already in pytorch models\detection folder (AppData\Local\Continuum\anaconda3\lib\site-packages\torchvision\models\detection\roi_heads.py). Without changing anything but only the path and hyperparameters in the cfg.py I still cannot make it run even I use the same version when you write the code. Sorry to bother you again.

**Comment by prajjwal1** [comment]
What the error is saying is that the data type of tensors inside target list is not of type `torch.int64`. This can be fixed by type casting the tensors to `torch.int64`. As you can see, this is an AssertionError. Please ensure that all the tensors inside targets are of type `torch.int64` and then it will work.
